assets/addon_browser/addons.conf

```diff
@@ -487,5 +487,14 @@
       "modrinth": "https://modrinth.com/mod/bluemap-sign-markers"
       "github": "https://github.com/tpwalke2/BlueMapSignMarkers"
     }
+  {
```

Comment:
Please add another closing curly bracket to close the previous item
They're currently unbalanced